.eslines.json

```diff
@@ -0,0 +1,8 @@
+{
+    "rulesNotToDowngrade": ["no-unused-vars"],
```

Comment:
The core idea of `eslines` is that it downgrades errors to warnings if they happen in lines not modified in the current branch. This array contains the rules that won't be downgraded no matter whether the line was modified or not. I tried to convey that idea in the [config section of the documentation](https://github.com/Automattic/eslines#config-file).

Different teams can have different use cases for this. Ours, and one that I'd recommend for every project, is to not downgrade `no-unused-vars`, as this is a case where modifying a line would cause a lint error on a different line - imports that become unused, for example. I could introduce this recommendation in the `eslines` docs.